integrations/microstream/cache/src/test/java/io/helidon/integrations/microstream/cache/CacheTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.integrations.microstream.cache;
+
+import java.nio.file.Path;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import io.helidon.config.ClasspathConfigSource;
+import io.helidon.config.Config;
+import io.helidon.config.ConfigSources;
+import one.microstream.cache.types.Cache;
+import one.microstream.cache.types.CacheConfiguration;
+
+public class CacheTest {
+
+	@TempDir
+	Path tempDir;
+
+	@Test
+	public void createCacheTest()
+	{

```

Comment:
Please use the same formatting as for the rest - e.g. opening braces on the same line as method declaration

Reply:
ok, fixed